packages/webpack-helpers-console-microfrontend/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@mia-platform/webpack-helpers-console-microfrontend",
```

Comment:
```suggestion
  "name": "@mia-platform/console-microfrontend-webpack-helpers",
```

Reply:
Helpers is correct? Is it this which developers will import, maybe something else? Also for vite

Reply:
You will use this library if you already use vite.
we had decided to use helpers to describe possible utilities to simplify the implementation of vite
Microfrontend side the developer will not use anything else

Reply:
name changed